cmd/environments/get.go

```diff
@@ -63,15 +64,25 @@ func runGet(
 			viper.GetString(cliflags.ProjectFlag),
 		)
 		if err != nil {
-			return err
+			output, err := output.CmdOutputResource(
```

Comment:
We handle all errors the same way.

Reply:
maybe we consider renaming this since method/the struct we dump these objects into since they're handling more than just `Resources` in the way that we're defining them elsewhere - e.g. it's also handling errors, config, not just `Resources` like flags, projects, environments, etc. not going to die on that hill though

Reply:
Renamed to `CmdOutputSingular` and `CmdOutputMultiple` to be more generic.